api/specs/webserver/scripts/Makefile

```diff
@@ -7,6 +7,7 @@ _check_venv_active:
 
 .PHONY: install
 install: _check_venv_active
+	@cd ./../../../../services/web/server && make install-dev && cd -
```

Comment:
Question and actually for both Makefiles. is it really needed to install the full service and why?

Reply:
Yes, tit is. You need some definitions that are define in the base services. It's true both in the case of storage and in the case of webserver.